### ISSUE TITLE
No empty line between annotation and the class/method/property...

### ISSUE THREAD

**Issue by BraisGabin** [pinterest/ktlint#688]
This sould be flagged: 

```kotlin
@Mockable

class MyCoolClass
```

And the correction should be:

```kotlin
@Mockable
class MyCoolClass
```

There should not be empty lines between an annotation and the thing that it's "annotating".

**Comment by AdamMTGreenberg** [comment]
#755 For review for the fix
